hash.go

```diff
@@ -0,0 +1,38 @@
+package wolfSSL
+
+// #cgo CFLAGS: -g -Wall -I/usr/include -I/usr/include/wolfssl
+// #cgo LDFLAGS: -L/usr/local/lib -lwolfssl -lm
+// #include <wolfssl/options.h>
+// #include <wolfssl/wolfcrypt/hash.h>
+import "C"
+import (
+    "unsafe"
+)
+
+const WC_MD5_DIGEST_SIZE = int(C.WC_MD5_DIGEST_SIZE)
+const WC_SHA_DIGEST_SIZE = int(C.WC_SHA_DIGEST_SIZE)
+const WC_SHA256_DIGEST_SIZE = int(C.WC_SHA256_DIGEST_SIZE)
+const WC_SHA384_DIGEST_SIZE = int(C.WC_SHA384_DIGEST_SIZE)
+const WC_SHA512_DIGEST_SIZE = int(C.WC_SHA512_DIGEST_SIZE)
+
+const WC_SHA256 = int(C.WC_SHA256)
+
+func Wc_Md5Hash(input []byte, inputSz int, output []byte) C.int {
```

Comment:
What happens if a user has one of these hash algorithms (or any feature really) compiled out at the native wolfSSL/wolfCrypt level?  For example `./configure --disable-md5`?